### PR TITLE
Pipeline to pip

### PR DESCRIPTION
**Pull request by Hsankesara**
Solves #62 

Change Log 

- [x] #9 
- [x] #40 
- [x] support installing radar-pipeline using pip
- [x] read and execute local or remote config file
- [x] have a custom radar data reader module
- [x] #65 
- [x] #58

### CODE REVIEW

**Review by afolarin — APPROVED** [review]

**Comment by afolarin** [comment]
Nice work @Hsankesara :clap:  only some minor points, I think some of these we already discussed. In any case, we can push ahead with the first release now :+1:

**Comment by afolarin** [comment]
@Hsankesara -- I've added a couple of pages to the Wiki, for the core topics a Wiki page.
https://github.com/RADAR-base/radarpipeline/wiki/Pipeline-Core-Topics  -- we'll need to think about how to build this in a sensible way

File: config.yaml.template [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    data_type: mock # couldbe mock, local, sftp, s3
+    config:
+        ## In case of sftp, use the following format
+        # sftp_host:
+        # sftp_source_path:
+        # sftp_username:
+        # sftp_private_key:
+        # sftp_target_path:
+        ## In case of s3, use the following format
+        # aws_access_key_id:
```
[the remark this answers is not in the document]
Comment (Hsankesara) [reply config.yaml.template:16]: good idea, yeah I haven't integrated s3 protocol yet but will generalize it when I do so.

File: radarpipeline/io/connection.py
[the remark this answers is not in the document]
Comment (Hsankesara) [reply radarpipeline/io/connection.py:1]: Yes, I think that would be part of the next release. I looked into the method, and it seems relatively simple to implement.

**Pull request merged**